### PR TITLE
Filter locations using Get-ZLocation

### PR DESCRIPTION
**Pull request by theaquamarine**
Implements #41, ie `Get-ZLocation blah`, `Get-ZLocation -match foo,bar`.

`-Match` seemed as good a parameter name as any, I'm not attached to it.

### CODE REVIEW

File: ZLocation/ZLocation.Storage.psm1 [hunk 1]
```diff
@@ -1,5 +1,7 @@
 Set-StrictMode -Version Latest
 
+Import-Module (Join-Path $PSScriptRoot 'ZLocation.Search.psm1')
```
Comment (vors) [inline ZLocation/ZLocation.Storage.psm1]: Hmm, do we need this dependency here?
Separating code into 3 files was done on purpose, but maybe doesn't make much sense for such a small project. We can combine them into 1 big file probably.

But this import should not be needed as far as I see.

File: ZLocation.Tests/ZLocation.Storage.Tests.ps1 [hunk 2]
```diff
@@ -3,7 +3,7 @@ Import-Module $PSScriptRoot\..\ZLocation\ZLocation.Storage.psm1 -Force
 Describe 'ZLocation.Storage' {
 
     $originalCount = (Get-ZLocation).Count
-    
+
```
Comment (vors) [inline ZLocation.Tests/ZLocation.Storage.Tests.ps1]: Please, keep formatting changes in separate commits or better PRs - it makes reviews much easier.

File: ZLocation/ZLocation.Storage.psm1 (hunk 1 above)
Comment (theaquamarine) [reply ZLocation/ZLocation.Storage.psm1]: `Get-ZLocation` uses `Find-Matches`  from ZLocation.Search to do the matching now [ZLocation.Storage.psm1#L172](https://github.com/theaquamarine/ZLocation/blob/56276576b03d999dddc859d3ddab1c3953eb8cb8/ZLocation/ZLocation.Storage.psm1#L172)

The import's not ideal, but it's the best solution I could think of- the only place it could go that wouldn't require adding an import would be moving it into ZLocation.psm1- which might not be the worst thing in the world since it might have more in common with `Set-Zlocation` than `Get-ZService`?

Comment (vors) [reply ZLocation/ZLocation.Storage.psm1]: Oh, I see, I missed the Find-Matches call - though it's a built-in cmdlet. Ok that's good as is, maybe just combining everything into a single file in the future to simplify things would be a good move.
